spec/models/rushing_game_log_spec.rb

```diff
@@ -10,7 +10,7 @@
     it "should calculate yards per rushing attempt rounded to 2 decimal points" do
       rushing = RushingGameLog.new(rushing_attempts: 15, rushing_yards: 80)
 
-      expect(rushing.yards_per_carry).to eq(5.33)
+      expect(rushing.yards_per_carry).to eq(5.3)
```

Comment:
Review this test so the `it` description and expectation are consistent with what's needed. If it should be 2 decimal points, ensure the yards_per_carry is 2 dp, e.g. `5.33`